base-notebook/Dockerfile

```
@@ -5,6 +5,23 @@
 # https://hub.docker.com/_/ubuntu/?tab=tags&name=focal
 ARG ROOT_CONTAINER=ubuntu:focal
 
+## BEGIN MICROMAMBA WORKAROUND
+# For some reason, downloading micromamba with wget or curl leads to a segfault.
+# Download them in a separate stage with alpine.
+# Context: <https://github.com/jupyter/docker-stacks/pull/1594#issuecomment-1028426999>
+FROM alpine as download_micromamba_stage
+WORKDIR /tmp
+RUN set -x && \
+    arch=$(uname -m) && \
+    if [ "${arch}" = "x86_64" ]; then \
+        # Should be simpler, see <https://github.com/mamba-org/mamba/issues/1437>
+        arch="64"; \
+    fi && \
+    echo $arch && \
+    apk add --no-cache curl==7.80.0-r0 && \
+    curl -L "https://micromamba.snakepit.net/api/micromamba/linux-${arch}/latest" > micromamba.tar.bz2
```

```suggestion
    curl -L "https://micromamba.snakepit.net/api/micromamba/linux-${arch}/latest" -o /tmp/micromamba.tar.bz2
```

And I think WORKDIR can be avoided.

---

Also on hadolint's suggestion :joy:

---

Ok, let's keep it then

---

I'm making the commit now (to a separate branch to not disturb the CI). Using `WORKDIR` was hadolint, I'm going to use `-o`.

Ah! I didn't understand the causality here. Good point: no workdir needed, I think.